tests/runtime/test_grpc_server.py

```diff
@@ -67,7 +66,7 @@
 from tests.conftest import random_test_id
 from tests.fixtures import Fixtures
 from tests.runtime.conftest import runtime_grpc_test_server
-import caikit
+import caikit.interfaces.common
```

Comment:
I forget exactly why I added this, but I _think_ it was to fix a bug that got [fixed elsewhere](https://github.com/caikit/caikit/pull/272)